### ISSUE TITLE
Allow admins to manually input alt text for item images

### ISSUE THREAD

**Issue by evan-lauer** [DigitalCarleton/AccessibilityPlus#14]
We should allow admins to manually input alt-text for item images which overrides the chosen metadata element. 

Additionally, it might make sense to create an "alt-text enforcement mode" which requires all new items to have a specialized alt-text field, instead of just choosing a metadata element. Alt text should be rich, and specific to the image. An item description, for example, may not actually describe what the image shows. Sometimes an image might be very different from the description of the item it represents.

**Comment by apjmason** [comment]
I need more detail on this. Dublin Core extended includes an Alternative Text field, so the simplest is to change help text to strongly urge that route for customization. 

Enforcement mode would be nice, maybe coupled with the former? If admin checks enforcement box, the plugin requires DC Extended to be installed and validates that Alt Text field has a value before submit?

**Comment by evan-lauer** [comment]
Yes, I like that. In fact, it might even be arguable that DC Extended should always be installed, and alt-text enforcement defaults to true. It's good practice to strongly, strongly encourage "custom" alt text rather than adopting a value from another field.

But that's just food for thought. Maybe for now I'll try to just add an enforcement mode which requires DC Extended, but without that, the normal DC metadata method applies.
